package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "chess.js": "^1.0.0-beta.6",
```

Comment:
removed for now because it'll be easier to not have conflicting type names, but we should add it back later to do performance tests